templates/resources/vmaas_load_balancer_monitor.md.tmpl

```diff
@@ -0,0 +1,54 @@
+---
+layout: ""
+page_title: "hpegl_vmaas_load_balancer_monitor Resource - vmaas-terraform-resources"
+subcategory: {{ $arr := split .Name "_" }}"{{ index $arr 1 }}"
+description: |-
+  {{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# Resource hpegl_vmaas_load_balancer_monitor
+
+-> Compatible version >= 5.4.6
+
+{{ .Description | trimspace }}
+`hpegl_vmaas_load_balancer_monitor` resource supports NSX-T Load balancer Monitor creation.
+
+For creating an NSX-T Load balancer Monitor, use the following examples.
+
+## Example usage for creating NSX-T Load balancer Monitor for HTTP with all possible attributes
+
+-> For NSX-T Load balancer Monitor for HTTP creation, attribute monitorType value `LBHttpMonitorProfile` is applicable only HTTP creation.
+
+{{tffile "examples/resources/hpegl_vmaas_load_balancer_monitor/nsx_t_lb_http_monitor.tf"}}
+
+## Example usage for creating NSX-T Load balancer Monitor for HTTPS with all possible attributes
+
+-> Once NSX-T Load balancer Monitor for HTTPs created with `LBHttpsMonitorProfile`, we can not modify the attribute during the update operation and its disabled from UI.
+
+{{tffile "examples/resources/hpegl_vmaas_load_balancer_monitor/nsx_t_lb_https_monitor.tf"}}
+
+## Example usage for creating NSX-T Load balancer Monitor for ICMP with all possible attributes
+
+-> Once NSX-T Load balancer Monitor for ICMP created with `LBIcmpMonitorProfile`, we can not modify the attribute during the update operation and its disabled from UI.
+
+{{tffile "examples/resources/hpegl_vmaas_load_balancer_monitor/nsx_t_lb_icmp_monitor.tf"}}
+
+## Example usage for creating NSX-T Load balancer Monitor for Passive with all possible attributes
+
+-> Once NSX-T Load balancer Monitor for Passive created with `LBPassiveMonitorProfile`, we can not modify the attribute during the update operation and its disabled from UI.
```

Comment:
not needed, please remove these in other places as well

Reply:
removed